lib/three_scale/domain_substitution.rb

```diff
@@ -102,33 +102,14 @@ module Account
     # or some internal checks (See lib/routing_constraints.rb)
     # @return [String] the domain of the Account
     def internal_domain
-      self['domain']
+      domain
     end
 
     # This is an alias to _#admin_domain_ as _#admin_domain_ will be private.
```

Comment:
`_#admin_domain_ will be private.` => `_#admin_domain_ is private.`

will be => is

Reply:
But not really, we decided to keep it deprecated no to touch activerecord stuff, didn't we?